db/migrate/20210216132553_create_grades.rb

```diff
@@ -2,7 +2,7 @@ class CreateGrades < ActiveRecord::Migration[6.1]
   def change
     create_table :grades do |t|
       t.integer :number
-      t.string :group, limit: 1
+      t.string :letter, limit: 1
```

Comment:
don't change existing migrations, it won't do anything because this migration already run
create new one

Reply:
I was trying to down migration just for grade to change group to letter, but it didn't allow me. That is why i used other methods to make changes.